topic/timeseries/dask-weather-data-import.ipynb

```diff
@@ -7,10 +7,19 @@
    "source": [
     "# How to Build Time Series Applications in CrateDB\n",
     "\n",
-    "This notebook guides you through an example of how to import and work with\n",
+    "This notebook guides you through an example of how to batch import \n",
     "time series data in CrateDB. It uses Dask to import data into CrateDB.\n",
     "Dask is a framework to parallelize operations on pandas Dataframes.\n",
     "\n",
+    "## Important Note\n",
+    "If you are running this notebook on a (free) Google Colab environment, you \n",
+    "might not see the parallelized execution of Dask operations due to constrained\n",
+    "CPU availability.\n",
+    "\n",
+    "We therefore recommend to run this notebook either locally or on an environment\n",
+    "that provides sufficient CPU capacity to demonstrate the parallel execution of\n",
+    "pandas operations as well as write operations to CrateDB.\n",
```

Comment:
Users might be slightly confused when it's about parallel Dask first, and then it's suddenly about parallel pandas, while both might be true.
```suggestion
    "Dask operations as well as write operations to CrateDB.\n",
```